apps/developer-portal/src/constants/help-panel-constants.ts

```diff
@@ -63,7 +63,7 @@ export class HelpPanelConstants {
      * @constant
      * @type {string}
      */
-    public static readonly EDIT_APPLICATIONS_DOCS_KEY = "[\"Developer Portal\"].Applications[\"Edit Application\"]";
+    public static readonly EDIT_APPLICATIONS_DOCS_KEY = "[\"Admin Portal\"].Applications[\"Edit Application\"]";
```

Comment:
Add a TODO here. That we need to change it when the mkdocs.yml is fixed.

Reply:
Fixed with `4f3eb92`